src/Microsoft.IdentityModel.Tokens/RsaSecurityKey.cs

```diff
@@ -33,10 +33,10 @@ internal RsaSecurityKey(JsonWebKey webKey)
         /// <param name="rsaParameters"><see cref="RSAParameters"/></param>
         public RsaSecurityKey(RSAParameters rsaParameters)
         {
-            IntializeWithRsaParameters(rsaParameters);
+            InitializeWithRsaParameters(rsaParameters);
         }
 
-        internal void IntializeWithRsaParameters(RSAParameters rsaParameters)
+        internal void InitializeWithRsaParameters(RSAParameters rsaParameters)
```

Comment:
@westin-m Shouldn't this change have triggered the public API analyzer?

Reply:
we cannot take any spelling changes if they impact the public API.

Reply:
I had a similar thought, but this is internal. I double checked SAL and MISE and there aren't usages of this through friend assemblies